modules/docs/markdown/02.1-serialisation/01-serialisation.md

```diff
@@ -0,0 +1,118 @@
+  * [jsonName](https://smithy.io/2.0/spec/protocol-traits.html#jsonname-trait)
+  * [timestampFormat](https://smithy.io/2.0/spec/protocol-traits.html#timestampformat-trait)
+  * [sparse](https://smithy.io/2.0/spec/type-refinement-traits.html#sparse-trait)
+  * [required](https://smithy.io/2.0/spec/type-refinement-traits.html#required-trait)
+  * [default](https://smithy.io/2.0/spec/type-refinement-traits.html#default-value-serialization). It is worth noting that, by default, Smithy4s chooses to not serialise default values if the when the member is optional.
+* [alloy traits](https://github.com/disneystreaming/alloy/blob/main/docs/serialisation/json.md)
+
+
+### XML
+
+The `smithy4s-xml` module provides [fs2-data](https://fs2-data.gnieh.org/documentation/xml/) encoders/decoders that can read/write generated data-types from/to XML bytes/strings. It is provided at the following coordinates :
+
+```
+sbt : "com.disneystreaming.smithy4s"  % "smithy4s-xml" % "@VERSION@"
+mill : "com.disneystreaming.smithy4s::smithy4s-xml:@VERSION@"
+```
+
+The entrypoint for  `smithy4s.xml.Xml`. See below for example usage.
+
+```scala mdoc:reset
+import smithy4s.example.hello.Person
+
+import smithy4s.Blob
+import smithy4s.xml.Xml
+
+val personEncoder = Xml.encoders.fromSchema(Person.schema)
+val personXML = personEncoder.encode(Person(name = "John Doe")).toUTF8String
+
+val personDecoder = Xml.decoders.fromSchema(Person.schema)
+val maybePerson = personDecoder.decode(Blob(personXML))
+```
+
+By default, `smithy4s-xml` abides by the semantics of :
+
+* [official XML-related smithy traits](https://smithy.io/2.0/spec/protocol-traits.html#xml-bindings)
+
+### Protobuf
+
+The `smithy4s-protobuf` module provides [protocol-buffers](https://protobuf.dev/) codecs that can read/write generated data-types from protobuf-encoded bytes.
+
+```
+sbt : "com.disneystreaming.smithy4s"  % "smithy4s-protobuf" % "@VERSION@"
+mill : "com.disneystreaming.smithy4s::smithy4s-protobuf:@VERSION@"
+```
+
+The entrypoint for XML parsing/writing is `smithy4s.protobuf.Protobuf`. See below for example usage.
+
+```scala mdoc:reset
+import smithy4s.example.hello.Person
+import smithy4s.protobuf.Protobuf
+
+val personCodec = Protobuf.codecs.fromSchema(Person.schema)
+val personBytes = personCodec.writeBlob(Person(name = "John Doe"))
+val maybePerson = personCodec.readBlob(personBytes)
+```
+
+By default, `smithy4s-protobuf` abides by the semantics of :
+
+* [alloy protobuf traits](https://github.com/disneystreaming/alloy/blob/main/docs/serialisation/protobuf.md). These semantics are the exact same semantics that [smithy-translate](https://github.com/disneystreaming/smithy-translate) uses to translate smithy to protobuf. This implies that the Smithy4s protobuf codecs are compatible with the codecs of other protobuf tools, generated from the .proto files resulting from running smithy through smithy-translate. In short, Smithy4s and [ScalaPB](https://github.com/scalapb/ScalaPB) can talk to each other.
```

Comment:
**suggestion:** should we expand "smithy4s and scalapb can talk to each other" to "smithy4s, scalapb and other Protobuf implementations can talk to each other"? It's implied I think, but being explicit couldn't hurt.

Reply:
[e1bccc0](https://github.com/disneystreaming/smithy4s/pull/1455/commits/e1bccc0f340772c4356f2f77a893094e80f67ed5)